src/Common/nunit/ConsoleOptions.cs

```diff
@@ -396,6 +254,163 @@ private string ExpandToFullPath(string path)
 #endif
         }
 
+        private void ConfigureOptions()
+        {
+            // NOTE: The order in which patterns are added
+            // determines the display order for the help.
+
+            // Old Options no longer supported:
+            //   fixture
+            //   xmlConsole
+            //   noshadow
+            //   nothread
+            //   nodots
+
+            // Options to be added:
+            //   teamcity
+            //   workers
```

Comment:
@NikolayPianikov You could remove those three lines starting // Options to be added online or leave them in as is convenient. I'll be back in this code for another issue pretty quick and can fix the comments then.